Frameworks/CoreFoundationAdditions/_NSCFTemporaryRootObject.mm

```diff
@@ -16,4 +16,8 @@
 #include "_NSCFTemporaryRootObject.h"
 
 @implementation _NSCFTemporaryRootObject
+- (void)doesNotRecognizeSelector:(SEL)selector {
+    // According to the reference platform documentation, this method must never return.
+    abort();
```

Comment:
abort doesn't translate to watson dump, so we wont know the cases where this is hit.  Perhaps dereference NULL to 'gracefully' crash if there is such a thing.

---
In reply to: [71574896](https://github.com/Microsoft/WinObjC/pull/671#discussion_r71574896) [](ancestors = 71574896,71574472)